server/cmd/mmctl/commands/post.go

```diff
@@ -55,9 +65,12 @@ func init() {
 	PostListCmd.Flags().BoolP("follow", "f", false, "Output appended data as new messages are posted to the channel")
 	PostListCmd.Flags().StringP("since", "s", "", "List messages posted after a certain time (ISO 8601)")
 
+	PostDeleteCmd.Flags().Bool("confirm", false, "Confirm you really want to delete the post and a DB backup has been performed")
```

Comment:
@BenCookie95 We did talk offline about adding a `--permanent` flag that enables the hard deletion and, by default, only soft deleting the post. Is that still a proposal that you want to implement?